kahuna/public/js/components/gr-tooltip/gr-tooltip.js

```diff
@@ -20,6 +18,16 @@ tooltip.directive('grTooltip', ['onValChange', function (onValChange) {
                     element.attr('data-title', newTooltip);
                 }));
             }
+
+            element.bind('mouseenter', () => {
+                element.addClass('titip-default')
+                    .addClass(`titip-${position}`);
+
+                $timeout(() => {
+                    element.removeClass('titip-default')
+                        .removeClass(`titip-${position}`);
+                }, 1500);
```

Comment:
You may need to cancel the `$timeout` if the directive gets `$destroy`ed as the DOM may have been torn down, in which case not sure if the above will work or not?